source/features/swap-branches-on-compare.tsx

```diff
@@ -32,6 +32,7 @@ void features.add(import.meta.url, {
 	exclude: [
 		// Disable on Two-dot Git diff comparison #4453
 		() => /\.\.+/.exec(location.pathname)?.[0]!.length === 2,
```

Comment:
Could you also extract this to the top of the file? `const isTwoDotDiff = //`

Can be merged either way